crmd/notify.c

```diff
@@ -20,120 +20,501 @@
 #include <crm/crm.h>
 #include <crm/msg_xml.h>
 #include "notify.h"
+#include "crmd_messages.h"
 
 char *notify_script = NULL;
 char *notify_target = NULL;
+GListPtr notify_list = NULL;
+
+typedef struct {
+    char *id;
+    char *path;
+    int timeout;
+    char *tstamp_format;
+    char *info;
```

Comment:
A local config-file would be a possibility. But would people spend that effort on their scripts?

The entire config being in pacemaker and visible through the usual tooling throughout the cluster is definitely appealing.

per-recipient timeout benefit I had in mind:
If it hangs for one recipient this would be killed way faster (number of recipients x faster) and further notification wouldn't be blocked. Further on we have parallel notification for all recipients without the script having to implement anything for that. But maybe having problems with it ;-) But that is a different story and would again lead us to serialization of certain streams of alerts.

Any pointer to that instance attribute sets you mentioned?